docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -49,7 +47,7 @@ You can analyze more than one application at a time against more than one transf
 * Azure
 ** Azure App Service
 ** Azure Kubernetes Service
-
+he only reason to add this feature to other Considerations doc is if this feature is something new and majo 
```

Comment:
I don't understand this sentence, I think it was intenteded to be a comment

Reply:
@abrugaro, heh, no idea, to be honest :-D I'm just reviewing and fixing what has already been in the source files for some time. But it really seems to be a comment....Maybe I somehow just missed removing it -> Will be fixed, thank you very much!